Mobile Buy SDK/Mobile Buy SDK/Utils/BUYApplePayHelpers.m

```diff
@@ -51,11 +51,6 @@ @interface BUYApplePayHelpers ()
 
 @implementation BUYApplePayHelpers
 
-- (instancetype)initWithClient:(BUYClient *)client checkout:(BUYCheckout *)checkout
-{
-	return [self initWithClient:client checkout:checkout shop:nil];
-}
-
 - (instancetype)initWithClient:(BUYClient *)client checkout:(BUYCheckout *)checkout shop:(BUYShop *)shop
```

Comment:
We can still set the checkout via instance variable. But it shouldn't be mutable from the outside.